src/com/jwetherell/algorithms/strings/StringFunctions.java

```diff
@@ -14,13 +14,21 @@ public class StringFunctions {
 
     public static final String reverseWithStringConcat(String string) {
         String output = new String();
+       
```

Comment:
Simple if statement that checks if the string entered into the function is null. If the string is null, then the function will return an empty string